add ability to hide/show time in menubar

nishantapatil3 opened the pull request:
Love your app!

This would help other users
- Adds ability to hide time in menubar

## Screenshot
<img width="276" height="768" alt="Screenshot 2025-12-03 at 10 37 43 AM" src="https://github.com/user-attachments/assets/67d650a6-1345-48d2-8841-521000c8fb2e" />

minhtc commented:
Thanks for the contribution! 🙌

It looks like the PR has conflicts because the codebase has been updated since your branch was created.  
Could you pull the latest changes from `main` into your branch and resolve the conflicts?  
After updating, feel free to push again. I’ll review it right away.

Thanks again for helping improve TimeMate!

Commit:
feat: add ability to show/hide time in menubar

Signed-off-by: Nishant Patil <nishantpatil@Nishants-MacBook-Air.local>

nishantapatil3 commented:
> Thanks for the contribution! 🙌
> 
> It looks like the PR has conflicts because the codebase has been updated since your branch was created. Could you pull the latest changes from `main` into your branch and resolve the conflicts? After updating, feel free to push again. I’ll review it right away.
> 
> Thanks again for helping improve TimeMate!

Of course! rebased and removed xcodeproj config changes
